utils/docker/vcluster/daos-base/el8/Dockerfile

```
@@ -0,0 +1,92 @@
+#                       a random value such as the date of day shall be given.
+# - DAOS_AUTH:          Enable DAOS authentification when set to "yes" (default "no")
+# - DAOS_REPOS:         List of repos needed to install DAOS (default
+#                       "https://packages.daos.io/v2.0")
+# - DAOS_GPG_KEYS:      List of GPG keys associated with DAOS repos (default
+#                       "https://packages.daos.io/RPM-GPG-KEY")
+# - DAOS_REPOS_NOAUTH:  List of repos to use without GPG authentification (default "")
+
+# Pull base image
+ARG	RHEL_BASE_IMAGE=centos
+ARG	RHEL_BASE_VERSION=8
+FROM	$RHEL_BASE_IMAGE:$RHEL_BASE_VERSION
+LABEL	maintainer="daos@daos.groups.io"
+
+# Configure systemd: more details could be found at following URL:
+# https://markandruth.co.uk/2020/10/10/running-systemd-inside-a-centos-8-docker-container
+# XXX FIXME XXX Should be removed in production with application dedicated entry point
+VOLUME	[ "/sys/fs/cgroup" ]
+RUN	systemctl mask systemd-remount-fs.service graphical.target kdump.service &&                \
+	systemctl unmask dev-hugepages.mount &&                                                    \
+	pushd /lib/systemd/system/sysinit.target.wants &&                                          \
+	for i in * ; do                                                                            \
+		[[ $i == systemd-tmpfiles-setup.service ]] || rm -f $i ;                           \
+	done &&                                                                                    \
+	popd &&                                                                                    \
+	rm -f /lib/systemd/system/multi-user.target.wants/* &&                                     \
+	rm -f /etc/systemd/system/*.wants/* &&                                                     \
+	rm -f /lib/systemd/system/local-fs.target.wants/* &&                                       \
+	rm -f /lib/systemd/system/sockets.target.wants/*udev* &&                                   \
+	rm -f /lib/systemd/system/sockets.target.wants/*initctl* &&                                \
+	rm -f /lib/systemd/system/basic.target.wants/* &&                                          \
+	rm -f /lib/systemd/system/anaconda.target.wants/*
+STOPSIGNAL SIGRTMIN+3
+ENTRYPOINT [ "/sbin/init" ]
+
+# Base configuration of dnf
+RUN	dnf clean all &&                                                                           \
+	dnf makecache &&                                                                           \
+	dnf --assumeyes install dnf-plugins-core &&                                                \
+	dnf config-manager --save --setopt=assumeyes=True &&                                       \
+	dnf config-manager --save --setopt=fastestmirror=True &&                                   \
+	dnf config-manager --set-enabled powertools &&                                             \
+	dnf install epel-release
+
+# Update system
+RUN	dnf update
```

You'll have fewer and slightly smaller images if you combine these.  What we've tried to do elsewhere is to leave the dnf cache empty between layers, assuming that's still the case here then the dnf clean all on L46 would be redundant as well.
```suggestion
	dnf install epel-release && \
        dnf update && \
        dnf clean all
```

---

The idea was to have a dedicate layer for the update to easily identify what was updated.
However, I am not sure that it will be really used => i am going to merge it as you suggest.

---

fixed